### ISSUE TITLE
BigQuery resource not found: <project-name>.SAFE.INT64

### ISSUE THREAD

**Issue by dion-gdn** [GoogleCloudPlatform/zetasql-toolkit#82]
Hi,

I got this error:
`com.google.zetasql.toolkit.catalog.bigquery.exceptions.BigQueryResourceNotFound: BigQuery resource not found: <project-name>.SAFE.INT64`

Here's the sample of my query:
```sql
SELECT
  ... // (omitted)
  SAFE.STRING(data.foo) AS foo,
  SAFE.INT64(data.bar) AS bar,
  ...
```

Where `data` type is [`JSON`](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#json_type).

Any idea what's wrong? Thanks

**Comment by ppaglilla** [comment]
Fixed by b449bde4e33d54d7c00fc9eb1713a965c8c9edcd. Will be included in release `0.5.3` in the next few days.

**Comment by gdncomm-dion-saputra** [comment]
hi @ppaglilla, are you sure this is fixed? I tested against the same query using branch version/v0.5.3 but still got the same error.

I think the error is thrown during getRoutinesOfType from BigQuery API:
https://github.com/GoogleCloudPlatform/zetasql-toolkit/blob/ca4445014dcfff8e0180dd8132beb4b54068a7cf/zetasql-toolkit-bigquery/src/main/java/com/google/zetasql/toolkit/catalog/bigquery/BigQueryAPIResourceProvider.java#L525-L531

During debugging before getting error:
![image](https://github.com/user-attachments/assets/6053a7b7-908f-4aea-ba17-b0d4f77270ee)
